File: angular/src/app/services/dashboard-data.service.ts
import { Injectable } from '@angular/core';
import { HttpClient } from '@angular/common/http';
import { Observable } from 'rxjs';

@Injectable({
  providedIn: 'root'
})
export class DashboardDataService {
  constructor(private http: HttpClient) { }

  public getAllTeamMembers(): Observable<any> {
    return this.http.get(`https://excel2json.io/api/share/d2d94130-e6aa-4387-437e-08da496bf5f2`);
  }
}

File: blazor-server/Template4o/Data/DashboardDataService.cs
using System.Net.Http.Json;

namespace Template4o.DashboardData
{
    public class DashboardDataService
    {
        private readonly HttpClient http;

        public DashboardDataService(HttpClient http)
        {
            this.http = http;
        }

        public async Task<AllTeamMembersType[]?> GetAllTeamMembers()
        {
            using HttpResponseMessage response = await http.GetAsync("https://excel2json.io/api/share/d2d94130-e6aa-4387-437e-08da496bf5f2").ConfigureAwait(false);
            if (response.IsSuccessStatusCode)
            {
                return await response.Content.ReadFromJsonAsync<AllTeamMembersType[]>().ConfigureAwait(false);
            }

            return null;
        }
    }
}